### PR TITLE
refactor(WidgetFrame): change to script setup

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

### CODE REVIEW

File: src/services/dashboards/widgets/_components/WidgetFrame.vue [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <template>
     <div class="widget-frame"
-         :class="{ full: isFull, 'edit-mode': editMode }"
-         :style="{ width: width ? `${width}px` : '100%' }"
+         :class="{ full: props.isFull, 'edit-mode': props.editMode }"
+         :style="{ width: props.width ? `${props.width}px` : '100%' }"
```
Comment (piggggggggy) [inline src/services/dashboards/widgets/_components/WidgetFrame.vue:4]: Can't you use it directly without accessing it with props?
```suggestion
         :class="{ full: isFull, 'edit-mode': editMode }"
         :style="{ width: width ? `${width}px` : '100%' }"
```

**Review by piggggggggy — APPROVED** [review]
LGTM!

I left single question!

**Review by sulmoJ — APPROVED** [review]
LGTM!!
Thank you for the refactoring!!

File: src/services/dashboards/widgets/_components/WidgetFrame.vue (hunk 1 above)
Comment (yuda110) [reply src/services/dashboards/widgets/_components/WidgetFrame.vue]: We can use like [this](https://sfc.vuejs.org/#eNqtUstugzAQ/JWVL3kIsHpFJFLVP+jZF0qWxBF+yDa0EuLfuzaEVImUUzng2d3xrGftkb1bWww9spJVvnHShqPQUlnjAozgsM3oVzdBDphBMJ/Y+gxO2EqNH4ZoGnWACVpnFGxIZyO00AB8H//4k3SIXvddWv9u246RA+Ax9Ha7gyUEaIz2AXyoA8JhbX+jx0/5cwmbC3ad2WT3tK4VUl73hlBPR5nT0+6GHLVy+t4JoCiK2dU29dutatMM0jIL7LnQFV+HRMGMZwP/MLZXvp8cP3mNh+Q8zryTjQwROPRemlV5jCpZ2km9D8u5FuOP3gIq21GBIoDq8nYc03aYJiA0a0wVp0IiSG37AEOuzAm7g2BEFYxKFV+FWMbmCeWqtsXVG02PLpkTS8ELVt4uRzAaS4wFu4Rgfcm5b5v4VK++MO7MCRWu10EqLNCr/MuZb4+OhAVbbjFpcEoO6HKH+oQO3SvNB+qTbnoMNGo2/QKgIhXN)!
But I prefer `state.xxx` or `props.xxx` in <script setup> case...

**Pull request merged**